sdk/node/test/accounts.js

```diff
@@ -0,0 +1,177 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const assert = require('assert')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const client = new chain.Client()
+const xAccountAlias = `x-${uuid.v4()}`
+const yAccountAlias = `y-${uuid.v4()}`
+
+let mockHsmKey
+
+describe('Accounts test', () => {
+
+  describe('Promise style', () => {
+
+    before('set up API objects', () => {
+
+      // Key and account creation
+      return client.mockHsm.keys.create()
+      .then(key => {
+        mockHsmKey = key
+
+        client.accounts.createBatch([
```

Comment:
I think you need a `return` here, or else the promise chain is unlikely to fail properly.